app/models/ability.rb

```diff
@@ -86,10 +86,8 @@ def signed_in(user)
       event.users.include?(user)
     end
 
-    # cannot create an event if program does not have open cfp
-    cannot [:new, :create], Event do |event|
-      user_inclusion = event.event_users.map { |event_user| event_user.user.id }.compact.include? user.id
-      !event.program.cfp_open? || !event.new_record? || !user_inclusion
+    can [:new, :create], Event do |event|
+      event.program.cfp_open? && event.new_record?
     end
```

Comment:
@sonalkr132 no, this is wrong
``
event.users.include?(user) ||  event.program.cfp_open? && event.new_record?
``
is not equivalent to what I used! Because I only use 
``
event.program.cfp_open? && event.new_record?
``
for :create and :new authorization.
But to be more clear, I changed manage to :update, :read, :destroy and the other one to :create!